assembl/static2/js/app/reducers/threadFilterReducer.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import {
+  RESET_THREAD_FILTER_DEFAULTS,
+  SET_THREAD_POSTS_ORDER,
+  THREAD_POSTS_MUST_BE_REFRESHED
+} from '../actions/threadFilterActions';
+import { defaultOrderPolicy } from '../components/debate/common/postsFilter/menu';
+
+type ThreadFilterState = {
+  postsOrderPolicy: PostsOrderPolicy,
+  postsMustBeRefreshed: boolean
+};
+
+const ThreadFilterReducer = (state: ThreadFilterState = {}, action: any) => {
+  const newState = { ...state };
+  switch (action.type) {
+  case '@@redux/INIT':
```

Comment:
You're catching the redux init in order set a default state? Why not just (state = initialState) and

const initialState = {
      postsOrderPolicy: defaultOrderPolicy,
      postsMustBeRefreshed: false
 };